addons/search-profile/style.css

```diff
@@ -1,10 +1,27 @@
-.tabs button:disabled {
+.user-tab {
+  display: grid !important;
+  align-items: end;
+  justify-items: center;
+}
+
+.user-tab:not(:disabled) img {
+  border-radius: 3px;
+}
```

Comment:
Dark mode currently breaks the icon.
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/1004ff56-28b8-43a5-ab3c-521471bda30b)
You can fix that by adding `filter: none`:
```suggestion
.user-tab:not(:disabled) img {
  border-radius: 3px;
  filter: none !important;
}
```